frontend/server/src/main/java/com/amazonaws/ml/mms/ModelServer.java

```diff
@@ -111,24 +129,25 @@ public void initModelStore() throws InvalidModelException, WorkerInitializationE
                         modelManager.updateModel(archive.getModelName(), 1, 1);
                     }
                 }
-            } else {
+            } else if (loadModels != null) {
                 String[] models = loadModels.split(",");
                 for (String model : models) {
-                    File modelFile = new File(modelStore, model);
-                    if (!modelFile.exists()) {
-                        if (!model.endsWith(".model")) {
-                            modelFile = new File(modelStore, model + ".model");
-                        }
-
-                        if (!model.endsWith(".mar")) {
-                            modelFile = new File(modelStore, model + ".mar");
-                        }
+                    String[] pair = model.split("=", 2);
+                    String modelName = null;
+                    String url;
+                    if (pair.length == 1) {
+                        url = pair[0];
+                    } else {
+                        modelName = pair[0];
```

Comment:
Should we validating urls here? in this file?

Reply:
This is handled in ModelArchive